src/json.cpp

```diff
@@ -1,4 +1,5 @@
 #include "json.h"
+#include "debug.h"
```

Comment:
This won't work when the `chkjson` target is build, it only uses "json.cpp" and "chkjson.cpp" - the static variables from "debug.cpp" are not available for it.